Feat/multiple user (#214)
deNBI/cloud-user-docs: wiki/portal/user_information.md

![setcopypublickey](img/profile_page.png)

> You have the possibility to view, copy and deposit your public ssh key. A set key is required for starting machines. Furthermore, you can only be added to machines by other members of your projects, which are started by them, if the key is set. 

Reviewer: ```suggestion
You have the possibility to view, copy and deposit your public ssh key. An uploaded public key is required for starting machines. 
```
The second sentence is for simplevm users, so I suggest to remove it.

Read more about it [here](../simple_vm/new_instance.md#3-grant-access-for-project-members).

> If you don't know how to generate a ssh key, you will find a link to the [instructions](../quickstart.md#generate-ssh-keys) on the page.

Reviewer: ```suggestion
If you don't know how to generate a ssh key,  you can read the [instructions](../quickstart.md#generate-ssh-keys) on our quickstart page.
```
